.github/actions/setup_dbt/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Setup dbt
+description: Install dependencies for dbt and configure it for use in CI.
+inputs:
+  role-to-assume:
+    description: AWS IAM role to assume when running dbt operations.
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Install dbt requirements
+      uses: ./.github/actions/install_dbt_requirements
+
+    - name: Load environment variables
+      uses: ./.github/actions/load_environment_variables
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v2
+      with:
+        role-to-assume: ${{ inputs.role-to-assume }}
+        aws-region: us-east-1
+
+    - name: Configure dbt environment
+      uses: ./.github/actions/configure_dbt_environment
```

Comment:
As of this PR, `setup_dbt` is the only action or workflow that's calling these nested actions. Arguably, this is a sign that we should get rid of the nested actions and move their step definitions into this action, but it also seems plausible that it might be useful for an action/workflow to pick and choose among these nested actions in the future for its own purpose, so for now I decided to maintain the factored-out actions and nest them in here.

Reply:
IMO this is simple enough while still preserving flexibility for the future. Good call.